### ISSUE TITLE
How to get the test set result from interaction dataset

### ISSUE THREAD

**Issue by aroundabout** [OpenDriveLab/HDGT#6]
Hi, @jiaxiaosong1002 ! 
I also try to submit the result to https://challenge.interaction-dataset.com/upload/.
But the scoreboard doesn't seem to be working properly.
I wanna to know how to get the result.
![image](https://github.com/OpenDriveLab/HDGT/assets/49650772/e793d8a2-b680-43e3-97db-e94950ad2605)
Thanks!

http://challenge.interaction-dataset.com/leader-board
![image](https://github.com/OpenDriveLab/HDGT/assets/49650772/b252f5df-f876-413f-aab8-ac0fd4f980cb)

**Comment by jiaxiaosong1002** [comment]
Hi,

Sorry. The reuslt of my test set INTERACTION results is on an older version of website (~3 years ago). It seems that they have made a major update. So, I do not know about their situation. Try to connect through their official website. Due to the internship change (~3 years) long long ago, I have no idea about it.

Thanks

**Comment by aroundabout** [comment]
ok, thank you for your reply

**Comment by aroundabout** [comment]
> Hi,
> 
> Sorry. The reuslt of my test set INTERACTION results is on an older version of website (~3 years ago). It seems that they have made a major update. So, I do not know about their situation. Try to connect through their official website. Due to the internship change (~3 years) long long ago, I have no idea about it.
> 
> Thanks

I found that your source code is uploaded last month. So it means the web page is still available last month? Have you ever mentioned?

**Comment by jiaxiaosong1002** [comment]
No. The online reuslts are submitted long long ago.

**Comment by aroundabout** [comment]
Ok, thank you very much,

**Comment by aroundabout** [comment]
![image](https://github.com/OpenDriveLab/HDGT/assets/49650772/e8592d17-ba0b-422a-b1ae-7807a0fec2f6)
Hi, the leader board can work now, but the scores of the other methods are lost. I wanna know which one you choose as the metric in your paper, "all" or "regular"? Or when you submit you result, there is no such catelgory.

**Comment by jiaxiaosong1002** [comment]
"All".  At that time, the default shown score is "all" while people could choose to see the score of "regular" or "gen". Basically, regular means the test cases are from the junction/intersection in the training set while "gen" means locations not in the training set. "All" is averaged all cases regardless of location.

**Comment by aroundabout** [comment]
thank you very much
